### PR TITLE
[#88] Configuration endpoint in s4e-web

### PR DESCRIPTION
**Pull request by tsatlawa**
- Create IConfigration interface
- Create init service to load configuration from backend
- Use APP_INITIALIZER to load config at startup
- Fix outsideBaseUrl to point to wms

### CODE REVIEW

File: s4e-web/src/app/app.module.ts [hunk 1]
```diff
@@ -19,14 +19,25 @@ import {AkitaNgRouterStoreModule} from '@datorama/akita-ng-router-store';
...
+import {ActivateModule} from './views/activate/activate.module';
+import {InitService} from './utils/initializer/init.service';
+import {InjectorModule} from './common/injector.module';
+import {ConfigProvider} from './utils/initializer/config.service';
+
 
 registerLocaleData(localePl, 'pl');
 
+
+export function initializeApp1(appInitService: InitService) {
```
Comment (jswk) [inline s4e-web/src/app/app.module.ts]: Why initializeApp1, with 1 at the end?

File: s4e-web/src/app/app.configuration.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import {JsonObject, JsonProperty} from 'json2typescript';
+
+export interface IConfiguration {
```
Comment (kmarszalek) [inline s4e-web/src/app/app.configuration.ts]: is there any convention with capital I before name?

File: s4e-web/src/app/app.module.ts (hunk 1 above)
Comment (tsatlawa) [reply s4e-web/src/app/app.module.ts]: My error, fixed.

File: s4e-web/src/app/app.configuration.ts (hunk 2 above)
Comment (tsatlawa) [reply s4e-web/src/app/app.configuration.ts]: To be honest I don't know. I see that @michal-szostak used `IConstants` so I used `IConfiguration`.

File: backend-development.env [hunk 3]
```diff
@@ -1,3 +1,4 @@
 SPRING_DATASOURCE_URL=jdbc:postgresql://db:5432/sat4envi
 SPRING_PROFILES_ACTIVE=development
 GEOSERVER_BASEURL=http://geoserver:8080/geoserver/rest
+GEOSERVER_OUTSIDE_BASEURL=http:://localhost:8080/geoserver/wms
```
[the remark this answers is not in the document]
Comment (tsatlawa) [reply backend-development.env]: Done

**Pull request merged**